modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -21,18 +22,29 @@ import {SidebarContext} from '../sidebar/SidebarContext.es';
 export default ({data, field, totalEntries, type}) => {
 	const {toggleSidebar} = useContext(SidebarContext);
 
+	const checkType = (field, type) => {
+		switch (type) {
+			case 'color':
+				return <Color hexColor={field} />;
+			case 'date':
+				return formatDate(field);
+			default:
+				return field;
+		}
+	};
+
+	const formatDate = (field) => {
+		const locale = themeDisplay.getLanguageId().split('_', 1).join('');
+
+		return moment(field).locale(locale).format('L');
```

Comment:
@alinedoleron, does `moment` handle internally `undefined` parameters, like `field` and `locale`?

Reply:
@natocesarrego, I've tested here and when moment receives a undefined value it returns an object like:
`Moment {_isAMomentObject: true, _isUTC: false, _pf: {…}, _locale: Locale, _d: Wed Jul 08 2020 17:26:21 GMT-0300 (Brasilia Standard Time), …}`. So I think it handles it.